packages/nextjs/src/config/define-config.ts

```diff
@@ -42,6 +42,7 @@ export const getNextFallbackConfig = (config?: SitecoreConfigInput): SitecoreCon
       process.env.GENERATE_STATIC_PATHS !== undefined
         ? process.env.GENERATE_STATIC_PATHS.toLowerCase() === 'true'
         : config?.generateStaticPaths ?? true,
+    editingInternalHostUrl: config?.editingInternalHostUrl || process.env.EDITING_INTERNAL_HOST_URL,
```

Comment:
According to our discussion, since it will be applied only for XMC, can we prefix it with "SITECORE"? 
```suggestion
    editingInternalHostUrl: config?.editingInternalHostUrl || process.env.SITECORE_INTERNAL_EDITING_HOST_URL,
```